### PR TITLE
fix(core): fixing accessibility of radio button component

### PR DESCRIPTION
**Pull request by web-dev23**
## Related Issue.
Part https://github.com/SAP/fundamental-ngx/issues/5805

## Description

Fix of accessibility issues of Radio Button component.

##### During Implementation
1. Visual Testing:
- [x] visual misalignments/updates
- [x] check Light/Dark/HCB/HCW themes
- [x] RTL/LTR - proper rendering and labeling
- [x] responsiveness(resize)
- [x] Content Density (Cozy/Compact/(Condensed))
- [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [x] Mouse vs. Keyboard support
- [x] Text Truncation
2. API and functional correctness
- [x] check for console logs (warnings, errors)
- [x] API boundary values
- [x] different combinations of components - free style
- [x] change the API values during testing
3. Documentation and Example validations
- [x] missing API documentation or it is not understandable
- [x] poor examples
- [x] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality
- [x] the commit message(s) follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

##### PR Review
- [x] visual misalignments/updates
- [x] check Light/Dark/HCB/HCW themes
- [x] RTL/LTR - proper rendering and labeling
- [x] responsiveness(resize)
- [x] Content Density (Cozy/Compact/(Condensed))
- [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [x] Mouse vs. Keyboard support
- [x] Text Truncation
2. API and functional correctness
- [x] check for console logs (warnings, errors)
- [x] API boundary values
- [x] different combinations of components - free style
- [x] change the API values during testing
3. Documentation and Example validations
- [x] missing API documentation or it is not understandable
- [x] poor examples
- [x] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

**Comment by web-dev23** [comment]
@puru-hk , unable to replicate Issue 4. 
44a5e2c4-ae23-453e-b098-1961ca299b33

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The below URL is not working.
https://fundamental-ngx-gh--pr6897-fix-radio-button-acc-97t05gg0.web.app/fundamental-ngx#/core/radio

The radio button associated label name is not read.
I can still able to reproduce the issue.
image


Reteet Details:
https://deploy-preview-6897--fundamental-ngx.netlify.app/#/core/radio
JAWS 2021  JAWS 2021.2107. 12 ILM 64bit for Windows 
Google Chrome: Version 95.0.4638.69 (Official Build) (64-bit)

**Comment by web-dev23** [comment]
> Hello Colleagues,
> 
> The below URL is not working. https://fundamental-ngx-gh--pr6897-fix-radio-button-acc-97t05gg0.web.app/fundamental-ngx#/core/radio
> 
> The radio button associated label name is not read. I can still able to reproduce the issue. image
> 
> Reteet Details: https://deploy-preview-6897--fundamental-ngx.netlify.app/#/core/radio JAWS 2021 JAWS 2021.2107. 12 ILM 64bit for Windows Google Chrome: Version 95.0.4638.69 (Official Build) (64-bit)

Unable to replicate the issue? for me labels (Option 1 etc) are read.
image

**Review by Betrozov — APPROVED** [review]

File: libs/core/src/lib/radio/radio-button/radio-button.component.html [hunk 1]
```diff
@@ -9,7 +9,6 @@
     [attr.aria-label]="ariaLabel"
     [attr.aria-labelledby]="ariaLabelledBy"
     [attr.aria-describedby]="ariaDescribedBy"
-    [attr.aria-disabled]="disabled"
```
[the remark this answers is not in the document]
Comment (platon-rov) [reply libs/core/src/lib/radio/radio-button/radio-button.component.html:12]: Any update on this one?

**Review by platon-rov — COMMENTED** [review]
Any updates on this? Otherwise i think it can be closed until work will be continued on this pr.

**Pull request closed without merging**